docs/source/indepth_tutorial/mpi-evaluator.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>why use pickle here instead of the save_results from the workbench?</p><p> </p><p>Pickle has many drawbacks and should not be used for longer-term data storage or cross-platform storage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quaquel/EMAworkbench/pull/308/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Good point, just a habit. Updated to use `save_results`.